docs/dev-guide/api-ref.md

```diff
@@ -101,6 +101,12 @@ where `:channelId` is the `_id` property of the channel to delete.
 
 The response code will be `200` if successful.
 
+#### Manually Trigger Polling Channel
+
+'GET /channels/:channelId/trigger'
```

Comment:
I think this needs to be a `POST` as `GET` requests shouldn't have any side affects if hit multiple times. Perhaps we should switch it over?